src/python/pants/util/objects.py

```diff
@@ -316,6 +421,9 @@ def satisfied_by_type(self, obj_type):
     :rtype: bool
     """
 
+  # TODO: This method could be extended to allow for coercion -- we could change the interface to
```

Comment:
Let's remove the TODO; it's really specific detail for something which may or may not happen :)